lib/kwtool_web/templates/auth/sign_in.html.eex

```diff
@@ -0,0 +1,17 @@
+<main class="form-authentication">
+  <h1>Login</h1>
+  <form>
+    <div class="form-group">
+      <label for="email">Email</label>
+      <input type="email" id="email" class="form-control" placeholder="Email Address" required autofocus>
```

Comment:
Maybe using form helper here would be nice as later we might want to show some form changeset errors, we will eventually use form_for 💡

Reply:
yes, thanks for the suggestion. I will use the form helper next time in the frontend tickets.